### ISSUE TITLE
Could not download database of phigaro

### ISSUE THREAD

**Issue by quliping** [bobeobibo/phigaro#40]
Phigaro is a wonderful tool, but I could not download the database. The link 'http://www.download.ripcm.com/phigaro/' seems not work due to unknown reasons. I'm not sure whether the problem is from the server or my own network.  It seems that 'Wget' sent the request but the server did not respond. Google drive is also not available for me because of government policy restrictions in our country. Is there any other way to download the database? 

![image](https://user-images.githubusercontent.com/39583607/192128497-8298eee5-9fa7-4bdf-a9db-b5a1f420ca44.png)

![image](https://user-images.githubusercontent.com/39583607/192128600-90333f96-47a6-4bfc-b6ac-0725f3bf7881.png)

**Comment by apcamargo** [comment]
Same problem here.

**Comment by PollyTikhonova** [comment]
Hello, @quliping ! 
I am so sorry that you bumped into this problem. Could you name services which are not blocked in your country? So that I could upload data there while we solving server errors? Also, I believe it would help other your colleagues to have these files on available servers.
Or I may try to send them via email, but the files are big and I am not sure if email box won't load them to another unavailable cloud.

**Comment by PollyTikhonova** [comment]
UPD. Right now the address http://download.ripcm.com/phigaro/ is working correctly
Please, let me know if it does not work on your side.

**Comment by quliping** [comment]
> UPD. Right now the address http://download.ripcm.com/phigaro/ is working correctly Please, let me know if it does not work on your side.

Thanks for your reply. I tried again and this address worked correctly this time. I have downloaded the database files successfully.  It means that the website 'http://download.ripcm.com/phigaro/' was not blocked by our country, and it is more like the sever doesn't work occasionally or the network fluctuation makes this address impossible to connect occasionally.  
Whatever, I got the database finally and phigaro works well. 

I will cite this software in my article. Thank you for your dedication to this software.

Sincerely
